tests/perf/test_perf_run.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import absolute_import, division, print_function
+
+import os
+import os.path
+import sys
+import subprocess
+
+import pytest
+
+
+PYTEST_PATH = (os.path.abspath(pytest.__file__.rstrip("oc"))
+               .replace("$py.class", ".py"))
+
+
+@pytest.fixture
+def popen(request):
+    env = os.environ.copy()
+    cwd = os.getcwd()
+    pythonpath = [cwd]
+    if env.get('PYTHONPATH'):
+        pythonpath.append(env['PYTHONPATH'])
+    env['PYTHONPATH'] = os.pathsep.join(pythonpath)
+
+    def popen_wait(*args, **kwargs):
+        __tracebackhide__ = True
+
+        args = [str(arg) for arg in args]
+        kwargs['env'] = dict(env, **kwargs.get('env', {}))
+
+        print('Running', ' '.join(args))
+        ret = subprocess.Popen(args, **kwargs).wait()
+        assert ret == 0
+
+    return popen_wait
+
+
+@pytest.fixture
+def color(pytestconfig):
+    reporter = pytestconfig.pluginmanager.getplugin('terminalreporter')
+    try:
+        return 'yes' if reporter.writer.hasmarkup else 'no'
+    except AttributeError:
+        return pytestconfig.option.color
+
+
+@pytest.fixture
+def verbosity(pytestconfig):
+    v = pytestconfig.option.verbose or -1
+    if v < 0:
+        return 'q' * (-v)
+    else:
+        return 'v' * v
```

Comment:
What do you think about solving this with a ternary operator like in the ```color``` function? The variable ```v``` can be reused obviously.